app/models/user.rb

```diff
@@ -26,4 +28,17 @@ def lock!
   def unlock!
     update(locked: false)
   end
+
+  def acceptable_avatar
+    return unless avatar.attached?
+  
+    unless avatar.byte_size <= 1.megabyte
+      errors.add(:avatar, "is too big")
+    end
```

Comment:
move it to private method

Reply:
divided validation into two separate methods